src/app/api/v1/relay/getRelayStatus.ts

```diff
@@ -0,0 +1,26 @@
+import { getPublicClient } from "@/lib/viem";
+import { cache } from "react";
+import { formatEther } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+
+const SPONSOR_PRIVATE_KEY = process.env.SPONSOR_PRIVATE_KEY;
+const GAS_COST = 0.001108297;
+
+async function getRelayStatus() {
+  if (!SPONSOR_PRIVATE_KEY) {
+    throw new Error("SPONSOR_PRIVATE_KEY is not set");
+  }
+  const publicClient = getPublicClient();
+  const account = privateKeyToAccount(SPONSOR_PRIVATE_KEY as `0x${string}`);
+
+  const balance = await publicClient.getBalance({
+    address: account.address,
+  });
+
+  return {
+    balance: Number(formatEther(balance)),
+    remaining_votes: Math.floor(Number(formatEther(balance)) / GAS_COST),
```

Comment:
same with here -- remaining_votes makes sense in the context of voting but how about delegations?